ui/src/app/pages/admin/admin-institution-edit.spec.tsx

```diff
@@ -44,39 +45,79 @@ describe('AdminInstitutionEditSpec', () => {
       .toContain('Display name must be 80 characters or less');
   });
 
-  it('should default DUA to Master', async() => {
+  it('should default DOMAINS to ADDRESSES in Registered Tier requirement', async() => {
```

Comment:
this text doesn't make sense

Reply:
I am unclear what is original test is trying to do. I would just delete this test because I don't feel it is useful.

Reply:
the original test checked that Add Institution chose a Master DUA as a default

Reply:
do we still need them then if we know now all institution will not have null TierRequirement enum in API?